tango_simlib/tango_sim_generator.py

```diff
@@ -402,9 +411,12 @@ def initialize_dynamic_attributes(self):
                     attr_props = UserDefaultAttrProp()
                     for prop in meta_data.keys():
                         attr_prop_setter = getattr(attr_props, "set_" + prop, None)
+                        # TODO (MTO): Add exception handling
+                        # Code silently breaks
```

Comment:
Silent problems are bad.  Hard to debug.  I take there isn't an easy way to make the failure loud?  It would be fine if a devices fails to start up due to invalid input.